src/theme/components/multiSelectMenuButton.ts

```diff
@@ -5,10 +5,11 @@ const MultiSelectMenuButton = {
     "selectedItemsCountButton",
     "toggleIcon",
   ],
-  baseStyle: ({ isOpen = false }) => ({
+  baseStyle: ({ hasSelectedItems = false, isOpen = false }) => ({
     buttonLabel: {
       justifyContent: "flex-start",
       overflow: "hidden",
+      marginLeft: hasSelectedItems ? "50px" : "0",
```

Comment:
Now that the selected items button is not displacing the button label text, we are losing the animation on the label text.  I thought the `transition` attribute was built into the CSS for all elements, but it looks like that is not the case and we’ll need to add the following for `buttonLabel`.

`transition: "margin 150ms ease-out”`

But that should only be applied when the first checkbox is selected.  When clearing the checkboxes, the selected items button disappears immediately, so the marign on the label text should not transition from 50 back to 0.  It should jump immediately from one to the next.  I am not sure which variable would indicate that action.